sync-data-to-another-database/plugin/lib/services/exporter.js

```diff
@@ -25,18 +25,18 @@ class Exporter {
     return this.client
       .connect()
       .then(() => {
-        this.context.log.info('[kuzzle-plugin-export-cassandra] Cassandra client connected');
+        this.context.log.info('[kuzzle-plugin-sync-cassandra] Cassandra client connected');
         return Promise.resolve(true);
       })
       .catch(error => {
         if (this.config.retryCount <= 0) {
-          return Promise.reject(new this.context.errors.InternalError(`[kuzzle-plugin-export-cassandra] Unable to connect the client : ${error.message}`));
+          return Promise.reject(new this.context.errors.InternalError(`[kuzzle-plugin-sync-cassandra] Unable to connect the client : ${error.message}`));
```

Comment:
Nitpicking: you may use `throw` inside a promise chain to return a reject promise. This simplify the syntax a bit.

Moreover, I think that a [ExternalServiceError](https://docs.kuzzle.io/api-documentation/errors/#externalserviceerror) is more suited than a [InternalError](https://docs.kuzzle.io/api-documentation/errors/#internalerror):

```
throw new this.context.errors.ExternalServiceError(`[kuzzle-plugin-sync-cassandra] Unable to connect the client : ${error.message}`);
```